spec/s3_meta_sync_spec.rb

```diff
@@ -345,7 +353,7 @@ def call(*args)
   end
 
   describe "CLI" do
-    let(:params) { "--key #{config[:key]} --secret #{config[:secret]} --region #{config[:region]}" }
+    let(:params) { "--key #{config[:key]} --secret #{config[:secret]} " }
```

Comment:
so the download does not work when it's in another region ? ... why does it not blow up our eu files then ?